src/assets/wise5/components/dialogGuidance/edit-dialog-guidance-feedback-rules/edit-dialog-guidance-feedback-rules.component.spec.ts

```diff
@@ -89,3 +100,43 @@ function expectFeedbackExpressions(expectedExpressionsInOrder: string[]): void {
     expect(component.feedbackRules[i].expression).toEqual(expectedExpressionsInOrder[i]);
   }
 }
+
+function createNewFeedbackRule() {
+  describe('createNewFeedbackRule', () => {
+    it('should create new feedback rule version 1', () => {
+      component.version = 1;
+      component.feedbackRules = [];
+      const feedbackRule = component.createNewFeedbackRule();
+      expect(typeof feedbackRule.feedback).toEqual('string');
+    });
+    it('should create new feedback rule version 2', () => {
+      component.feedbackRules = [];
+      const feedbackRule = component.createNewFeedbackRule();
+      expect(Array.isArray(feedbackRule.feedback)).toBeTruthy();
+    });
+  });
+}
```

Comment:
Make createNewFeedbackRule() private (since it's only being called by addNewRule()), so we should test this function indirectly (through addNewRule())